build-cluster.sh

```diff
@@ -6,6 +6,12 @@ if [ -z "$NCOMPUTES" ] || [ "$NCOMPUTES" -lt 1 ] || [ "$NCOMPUTES" -gt 10 ]; the
     exit 1
 fi
 
+PXEBOOT_ISO="$2"
+if [ ! -f "$PXEBOOT_ISO" ]; then
+    echo "You need to point to a PXEBOOT ISO from https://ipxe.org/download"
+    exit 1
+fi
+
```

Comment:
We can remove these checks, as having the target dependency will take care of it.
```suggestion
```